sdk/azidentity/go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/Azure/azure-sdk-for-go/sdk/azidentity
+
+replace github.com/Azure/azure-sdk-for-go/sdk/azcore => ../azcore
+
+go 1.13
+
+require (
```

Comment:
I think there are some entries here that aren't required.  Can you run `go mod tidy`?  It should clean up go.mod and go.sum.